### DISCUSSION TITLE
Images in my blog are getting replaced with this "data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 1280 720'%3E%3C/svg%3E"

### DISCUSSION THREAD

**Discussion by Cybrik7** [cotes2020/jekyll-theme-chirpy#1303] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I made a blog using jekyll theme with chirpy starter. everything was ok till last month , but now when I check my website , all the images are showing as blank. And when I click on that image and open it in new tab its showing this random url "data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 1280 720'%3E%3C/svg%3E" . I am using postimage for my image hosting.  

Here is the website if you wanna have a look at the issue - https://cybrik7.github.io/posts/thm-kenobi/

### Operations you have already tried

I already tried different image hosting sites to confirm whether it's an issue with my hosting postimage only , but it's not the case. It's happening with every hosting. Can anyone help?

**Comment by kungfux** [comment]
It looks like the issue was resolved, so closing it out. Feel free to reopen if needed.
